server/game/cards/01_SOR/ImperialInterceptor.ts

```diff
@@ -0,0 +1,31 @@
+import AbilityHelper from '../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../core/card/NonLeaderUnitCard';
+import { Location, RelativePlayer, WildcardCardType } from '../../core/Constants';
+
+export default class ImperialInterceptor extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9002021213',
+            internalName: 'imperial-interceptor'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may deal 3 damage to a space unit.',
+            optional: true,
+            targetResolvers: {
```

Comment:
Since there's only one target resolver, you can simplify it to use `targetResolver` (singular) and to have no name:

```
targetResolver: {
      cardTypeFilter: WildcardCardType.Unit,
      controller: RelativePlayer.Any,
      locationFilter: Location.SpaceArena,
      immediateEffect: AbilityHelper.immediateEffects.damage({ amount: 3 })
}
```

Then the `effectArgs` on line 26 would reference `context.target` instead of `context.targets.spaceUnit`

Reply:
Changed it